doc/sphinx/language/core/basic.rst

```diff
@@ -332,9 +332,9 @@ rest of the Coq manual: :term:`terms <term>` and :term:`types
 
    tactic
 
-     Tactics specify how to transform the current proof state as a
+     A :production:`tactic` specifies how to transform the current proof state as a
```

Comment:
We do this for a handful of syntactic elements that don't appear in the grammar.  For example, command, string, query_command, l1_tactic, ...  I don't they should appear in the glossary.  We can revisit/reconsider now or later if you wish.

Reply:
It's fine. It's just that I wasn't aware that `tactic` was a true syntactic element (as opposed to e.g., `l1_tactic` or `ltac_expr`).

Reply:
Maybe this should be handled a bit better, but let's address this later.  `string` should be defined as a `:production:` because it's a token, not a nonterminal.  The definition is in in a sentence, not a `:prod:`.  IIRC I introduced some of these before we add `:gdef:`.  `tactic` doesn't appear in orderedGrammar.  `l1_tactic` does, but it's empty and could as well be deleted.